src/ui/units/ql/containers/PaneMain/ScreenEditor/TabParams/DefaultValue.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+
+import {Xmark} from '@gravity-ui/icons';
+import {Button, Icon, TextInput} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+import {i18n} from 'i18n';
+
+import {QLParamType, QlConfigParam, TabParamsQA} from '../../../../../../../shared';
+import {getDatesFromValue} from '../../../../../../components/RelativeDatesPicker/utils';
+
+import {resolveAndFormatDate} from './utils';
+
+const b = block('ql-tab-params');
+
+interface DefaultValueProps {
+    param: QlConfigParam;
+    paramIndex: number;
+    onClickButtonEditParamValue: ({
+        value,
+        type,
+        onApply,
+    }: {
+        value: string;
+        type: QLParamType;
+        onApply: ({value}: {value: string}) => void;
+    }) => void;
+    onEditParam: ({param, paramIndex}: {param: QlConfigParam; paramIndex: number}) => void;
+    onClickButtonAddParamValue: ({paramIndex}: {paramIndex: number}) => void;
+    onClickButtonRemoveParamValue: ({
+        paramIndex,
+        valueIndex,
+    }: {
+        paramIndex: number;
+        valueIndex: number;
+    }) => void;
+}
+
+export const DefaultValue = ({
+    param,
+    paramIndex,
+    onClickButtonEditParamValue,
+    onEditParam,
+    onClickButtonAddParamValue,
+    onClickButtonRemoveParamValue,
+}: DefaultValueProps) => {
+    const paramIsInterval =
+        param.type === QLParamType.DateInterval || param.type === QLParamType.DatetimeInterval;
+
+    const paramIsDate = param.type === QLParamType.Date || param.type === QLParamType.Datetime;
+
+    if (
+        paramIsInterval &&
+        typeof param.defaultValue === 'object' &&
+        !Array.isArray(param.defaultValue) &&
+        param.defaultValue.from &&
+        param.defaultValue.to
```

Comment:
it seems all these checks are duplicated in the OverrideValue component? Is there any way to put them in a common place?